package.json

```diff
@@ -84,7 +84,7 @@
   "author": "Juan Cruz Viotti <juan@balena.io>",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.20.0"
+    "node": ">=10.20.0 <13.0.0"
```

Comment:
With this change, when using Node.js v14, `npm install` still succeeds but then when running the CLI the result is:
```
$ ./bin/balena version -a
------------------------------------------------------------------------------
Warning: Node version "v14.15.4" does not match required versions ">=10.20.0 <13.0.0".
This may cause unexpected behavior. To upgrade Node, visit:
https://nodejs.org/en/download/
------------------------------------------------------------------------------
balena-cli version "12.38.3"
Node.js version "14.15.4"
```

As question was whether to use `<13.0.0` or `<14.0.0`. We know the CLI works with Node v12, we know it does not work with Node v14, but we are not sure about Node v13. I think that a good principle would be _"it works unless we know it doesn't"_ (as this encourages people to test and report bugs), by which we should use `<14.0.0`. However, because this change will be disruptive to many users who've been using Node.js v14 for a while (we've allowed it for several months), it would be a shame if many users went through the trouble of downgrading to Node.js v13, only for us later finding that v13 also doesn't work, and disrupt users again.

Reply:
Also of potential interest:
* https://github.com/Homebrew/homebrew-core/pull/63674#discussion_r558329656
* https://aur.archlinux.org/cgit/aur.git/commit/?h=balena-cli&id=fc21a59bec910c9a92c0d172c645d092fe643bea